library/core/src/ffi/c_str.rs

```diff
@@ -885,7 +885,32 @@ mod verify {
             assert!(c_str.is_safe());
         }
     }
-  
+
+    // pub const fn from_bytes_with_nul(bytes: &[u8]) -> Result<&Self, FromBytesWithNulError>
+    #[kani::proof]
+    #[kani::unwind(17)]  
+    fn check_from_bytes_with_nul() {
```

Comment:
Is it possible to extend this harness to any combination of bytes up to a certain size?

Reply:
Sure, will do. Also note that it fails for bit sizes >= 32

Reply:
Why does it fail? Which error are you getting?